content/2017/1/11/typography.rst

```diff
@@ -25,6 +25,10 @@ after:
 How to Choose Engaging & Accessible Typography for Your Website Brand
 =====================================================================
 
+.. callmacro:: blog/macros.j2#byline
+   get_blog_entries_by_bird('kit')[0]
```

Comment:
Played with this a little; there are some details to the innards of the argument parsing that I'll need to sort out before I can get this approach to work. Punting on it for now.